extensions/amp-analytics/0.1/events.js

```diff
@@ -37,6 +39,8 @@ const DEFAULT_MAX_TIMER_LENGTH_SECONDS = 7200;
 const VARIABLE_DATA_ATTRIBUTE_KEY = /^vars(.+)/;
 const NO_UNLISTEN = function () {};
 const TAG = 'amp-analytics/events';
+const TIME_WAIT = 500;
```

Comment:
The name `TIME_WAIT` could apply to any sort of wait but it's used for something specific. Let's rename to `SESSION_DEBOUNCE_TIME_MS`.